packages/components/src/duotone-picker/utils.ts

```diff
@@ -78,10 +91,12 @@ export function getColorStopsFromColors( colors ) {
 /**
  * Convert a color stop array to an array colors.
  *
- * @param {Object[]} colorStops Color stop information.
+ * @param colorStops Color stop information.
  *
- * @return {string[]} CSS colors array.
+ * @return CSS colors array.
  */
-export function getColorsFromColorStops( colorStops = [] ) {
+export function getColorsFromColorStops(
+	colorStops: { position: number; color: string }[] = []
```

Comment:
@chad1008 there may also be some overlap with `CustomGradientPicker` around the color stop type

Reply:
Good catch, there is some interesting overlap, and some potentially confusing semantics.

In `CustomGradientPicker` we have two related constructs:
1. A `ColorStop` which is each of the specific colors of the gradient. With #48929, we will get type data for this from the `gradientParser` library's `DefinitelyTyped` data.
2. A `ControlPoint`, which is how our component places the UI handles to adjust the `ColorStops`.

The two are related, but shaped differently.

In `DuotonePicker`, it looks like there's some mixing/conflating of those terms. (not new to this PR). `getColorStopsFromColors`, for example, returns an object that is shaped that like a `ControlPoint` and not a `ColorStop`. The component also [uses that function to populate a variable named `controlPoints`](https://github.com/WordPress/gutenberg/blob/11295e10c929b0606191fbb2f24bdf7f369659d5/packages/components/src/duotone-picker/custom-duotone-bar.tsx#L24).

All of this to say, we might want to rename that function to `getControlPointsFromColors`, but functionally I don't think there's an issue. The correct type is being passed into `CustomGradientPicker`, and when I rebase the two PRs together, Storybook works just fine 👍

Reply:
With #48929 merged, we can now rebase and see how to best integrate `CustomGradientPicker`'s types into `Duotone`